### ISSUE TITLE
Read camtrap packages with camtrapR

### ISSUE THREAD

**Issue by niconoe** [tdwg/camtrap-dp#29]
In GitLab by @peterdesmet on Aug 25, 2020, 9:25

_This is more of an implementation issue, but I'm logging it here so we don't forget._

Users will likely want to use the [camtrapR package](https://jniedballa.github.io/camtrapR/) to analyse their data. We should provide functionality/tutorials to read a camtrap package and map it to the camtrapR data model. Data package reading functionality can probably be included by using [datapackage.r](https://github.com/frictionlessdata/datapackage-r).

I'm not a user of camtrapR myself, so I'm not very familiar with it. Jakub, any thoughts how how to provide this functionality?

**Comment by niconoe** [comment]
In GitLab by @kbubnicki on Aug 27, 2020, 15:58

Definitely this integration will be crucial for many people.

I have just sent an invitation to join us to Jurgen Niedballa, the main developer behind the camtrapR package. Lets see what he will say about this :)

**Comment by niconoe** [comment]
In GitLab by @jniedballa on Oct 1, 2020, 13:38

Hi all,
Jakub kindly introduced me to the project and we began discussing how to incorporate the new standard into camtrapR. There's still some open questions as long as the standard is not finalised, but I believe it can be done. I'll have a look at the project homepage here and see if I can provide some input for the standard. 
My first impression was that the standard is mostly reasonably similar to the camtrapR data model and can probably be translated. And while the devil is usually in the detail, I'm hopeful it can be done.
Best regards,
Jürgen

**Comment by niconoe** [comment]
In GitLab by @jimcasaer on Oct 1, 2020, 13:56

Thanks Jürgen - would really be nice if this would work out !
greetings
jim

Dr.ir. Jim Casaer

Senior Researcher
Research Group wildlife Management / OT  Faunabeheer en Exoten/
Research Institute for Nature and Forest / Instituut voor Natuur- en
Bosonderzoek



Havenlaan 88 bus 73,
1000 Brussel

jim.casaer@inbo.be
** 32 478 281 701
www.inbo.be
orcid : 0000-0001-6788-5876

**Comment by niconoe** [comment]
In GitLab by @jimcasaer on Oct 1, 2020, 13:57

Thanks Jürgen - would be really nice if this would work out ! 
greetings
jim

**Comment by peterdesmet** [comment]
In 2021 we started the "camtraptor" R package to support the use of Camtrap DP. It has a function to read Camtrap DP (https://inbo.github.io/camtraptor/reference/read_camtrap_dp.html) and can convert data for CamtrapR (https://inbo.github.io/camtraptor/reference/get_cam_op.html).

Whether this or all camtraptor functionality should be integrated in CamtrapR is up for discussion (see https://inbo.github.io/camtraptor/#camtraptor-vs-camtrapr-and-activity) and not part of this repository. Closing issue.
